### PR TITLE
Make it work without jQuery

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: app/instance-initializers/browser/head.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
   if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
 
   // clear fast booted head (if any)
-  Ember.$('meta[name="ember-cli-head-start"]')
-    .nextUntil('meta[name="ember-cli-head-end"] ~')
-    .addBack()
-    .remove();
+  let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
+  let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
+  if (startMeta && startMeta) {
```
Comment (rwjblue) [inline app/instance-initializers/browser/head.js]: This seems odd?  Should it be `if (startMeta && endMeta)` ?

Comment (cibernox) [inline app/instance-initializers/browser/head.js]: Indeed. Fixed.

**Comment by erkarl** [comment]
Looks good to me. As of this moment this is the only addon preventing me to migrate off jQuery completely.

**Comment by cibernox** [comment]
I still have a few addons to fix to go.

**Comment by rwjblue** [comment]
@cibernox - Sorry, I made a rebase conflict for you by landing the fastboot@1.0.0-rc support PR.  I think you will also have to update this in a few places now (since we have multiple copies of this file and use the correct one depending ember-cli version and such).

**Comment by cibernox** [comment]
@rwjblue Rebased and green.

**Comment by rwjblue** [comment]
ember-cli-head 0.2.2 published 🎉

**Pull request merged**